packages/framework-integration-tests/integration/cli/cli.command.integration.ts

```diff
@@ -1,34 +1,42 @@
 import path = require('path')
-import util = require('util')
 import { expect } from 'chai'
-import { readFileContent, writeFileContent } from '../helper/fileHelper'
-
-const exec = util.promisify(require('child_process').exec)
+import {
+  createSandboxProject,
+  loadFixture,
+  readFileContent,
+  removeFolders,
+  writeFileContent,
+} from '../helper/fileHelper'
+import { exec } from 'child-process-promise'
 
 const COMMAND_AUTH_PLACEHOLDER = "// Specify authorized roles here. Use 'all' to authorize anyone"
 
-const FILE_CHANGE_CART_COMMAND = 'src/commands/change-cart.ts'
-const FILE_CHANGE_CART_WITH_FIELDS_COMMAND = 'src/commands/change-cart-with-fields.ts'
+describe('Command', () => {
+  const SANDBOX_INTEGRATION_DIR = 'command-integration-sandbox'
+  const FILE_CHANGE_CART_COMMAND = `${SANDBOX_INTEGRATION_DIR}/src/commands/change-cart.ts`
+  const FILE_CHANGE_CART_WITH_FIELDS_COMMAND = `${SANDBOX_INTEGRATION_DIR}/src/commands/change-cart-with-fields.ts`
+
+  before(async () => {
+    createSandboxProject(SANDBOX_INTEGRATION_DIR)
+  })
 
-export const CLI_COMMAND_INTEGRATION_TEST_FILES: Array<string> = [
-  FILE_CHANGE_CART_COMMAND,
-  FILE_CHANGE_CART_WITH_FIELDS_COMMAND,
-]
+  after(() => {
+    removeFolders([SANDBOX_INTEGRATION_DIR])
+  })
 
-describe('Command', () => {
-  const cliPath = path.join('..', 'cli', 'bin', 'run')
+  const cliPath = path.join('..', '..', 'cli', 'bin', 'run')
```

Comment:
Nit-picky, it looks like the path segments never change in cli tests. Does it make sense to use a placeholder constant with the segments and add that const to the `fileHelper`?